### DISCUSSION TITLE
[Feature Request] How to support shadeless material rendering like sketchfab?

### DISCUSSION THREAD

**Discussion by wzc2021** [google/model-viewer#2671] [Ideas]
### Description
The current library seems doesn't support render shadeless material, like sketchfab or meshlab, there's shade None option, would it be suppoted in the near future or is it already available in somewhere? thanks.

**Comment by elalish** [comment]
We support glTF; I'm not aware of a glTF material extension called "Shadeless"? We do support the Unlit extension, if that's synonymous.
